webpack.config.js

```diff
@@ -67,14 +67,6 @@ module.exports = {
     filename: '[name].[contenthash].js',
     publicPath: '/',
   },
-  resolve: {
-    alias: {
-      'patched-scratch-blocks': path.resolve(
-        __dirname,
-        'src/scratch-patches/scratch-blocks.js'
-      ),
-    },
-  },
```

Comment:
I was originally planning to alias `scratch-blocks`, but that didn't work out and was a bit stupid. Then I changed it to `patched-scratch-blocks`, which doesn't make any sense. When looking over the Webpack config for this PR, I realized that it's actually pretty useless to have a local file aliased. So I removed it!

Reply:
Sounds good!